bin/crew

```diff
@@ -769,21 +771,23 @@ def prepare_package (destdir)
 
     # check for conflicts with other installed files
     puts "Checking for conflicts with files from installed packages..."
-    conflicts = []
-    @conflictscmd = "grep --exclude #{CREW_META_PATH}#{@pkg.name}.filelist \
-      -Fxf filelist #{CREW_META_PATH}*.filelist | tr ':' ' ' | \
-      sed 's,.filelist,,g' | sed 's,#{CREW_META_PATH},,g'"
-    conflicts << %x[#{@conflictscmd}].chomp.split(" ")
-    conflicts.reject!(&:empty?)
-    unless conflicts.empty?
+    conflicts = {}
+    File.readlines('filelist').each do |p|
+      next unless File.exist?(p.chomp)
+      Dir['/usr/local/etc/crew/meta/*.filelist'].each do |f|
+        pkgName = File.basename(f, '.filelist')
+        c = File.foreach(f).grep( Regexp.new('^' + p + '$') )[0]
+        conflicts[c] = pkgName if c
+      end
+
       puts "Unable to complete this build since there is a conflict with the same file in another package.".lightred
-      pp = ''
-      conflicts.each do |p, f|
-        puts "\n#{p} package conflicts below:".lightred if p != pp
-        puts f.lightred
+      pp = nil
+      conflicts.each_pair do |f, p|
+        puts nil, "#{p} package conflicts below:".lightred if p != pp
+        puts f.chomp.lightred
         pp = p
       end
-      abort unless ENV['CREW_CONFLICTS_ONLY_ADVISORY'] == '1'
+      abort unless ENV['CREW_CONFLICTS_ONLY_ADVISORY']
```

Comment:
Does this abort with `CREW_CONFLICTS_ONLY_ADVISORY=false` or `CREW_CONFLICTS_ONLY_ADVISORY='0'`?

Reply:
Leave the `== 1` check in. It's much simpler, and leaves it in env for when you want to grep to see what the relevant env variables are. It's much easier to do a `export CREW_CONFLICTS_ONLY_ADVISORY=0` or whatever when you want to go back to regular mode, and then flip back as necessary without having to look up the relevant env variable again.

Reply:
Reverted change